docs/schema/conditions-element-workflowinfo-elementaction4.md

```diff
@@ -164,7 +165,7 @@ If the schema defines specific requirements, such as **sequence**, **minOccurs**
 <td align="left"><p>optional</p></td>
 <td align="left"><p>Displayed in the rule designer sentence when two or more conditions are used in the same conditional branch, and when all conditions must be satisfied before workflow actions can be executed. The value is not case-sensitive.</p>
 <p>The default value is **and** (applies only to the English language version of SharePoint).</p>
-<p>**Example**: `<Conditions And="and">`</p></td>
+<p>**Example**: `&lt;Conditions And="and">`</p></td>
```

Comment:
Not a huge deal, but the repo owner for this repo is preferring an alternative fix for these types of issues - so a better fix may be "\<Conditions And="and"\>".